SEED-platform/seed
# EEEJ Analysis Functionality

#### What's this PR do?
New EEEJ Analysis 

#### How should this be manually tested?
Add a few buildings (must contain address_line_1, city/state or postal_code)
Select buildings -> Run an Analysis -> select the EEEJ Analysis
Ensure that the analysis is successful
Check that the census tract field identified is correct (can use https://geocoding.geo.census.gov/geocoder/geographies/onelineaddress?form to compare)
Check that the DAC designation is correct (can use https://screeningtool.geoplatform.gov/en/#3/33.47/-97.5 to compare)

#### What are the relevant tickets?
#4207

#### Screenshots (if appropriate)
![Screen Shot 2023-08-23 at 9 52 41 PM](https://github.com/SEED-platform/seed/assets/2205659/e7259355-3f6d-49e4-ac70-cad34ead1f50)
![Screen Shot 2023-08-23 at 9 52 47 PM](https://github.com/SEED-platform/seed/assets/2205659/25457aac-1a4a-4121-9163-25c93f8d97bd)
